plugins/app-backend/src/service/router.ts

```diff
@@ -220,7 +220,7 @@ export async function createRouter(
           req.method = 'GET';
           next('router');
         } else {
-          throw new Error('Invalid POST request to /');
+          throw new InputError('Invalid POST request to /');
```

Comment:
Slightly misleading to say that it was to `/`, on a star path, right?

Reply:
Yep good call, I pointed to "app-backend" as well since that's another bit that might not be obvious when encountering this error given the wildcard path: https://github.com/backstage/backstage/pull/26664/commits/1231e6ecbbe6f732033895fe4c4e742343917fa9